packages/core/utils/src/common/__tests__/partition-array.spec.ts

```diff
@@ -1,4 +1,4 @@
-import { partitionArray } from "../../../dist"
+import { partitionArray } from "../../index"
```

Comment:
The source code imports should not look for files in the builder output. This was caught after type checking test files